views/cypress/utils/resource-manager.js

```diff
@@ -43,3 +43,56 @@ export function selectUploadLocalAsset(fileName, pathToFile) {
             cy.getSettled(`#${resourcemgrId} li[data-file="/${fileName}"] .actions a.select`).click();
         });
 }
+/**
+ * Add/upload shared stimulus previously created in assets
+ * * @param {boolean} isCreatedAsset determines whether the asset is
+ * the one previously created (vs imported)
+ */
+export function selectUploadSharedStimulus(isCreatedAsset) {
```

Comment:
This is related to taoMediaManager should be put in https://github.com/oat-sa/extension-tao-mediamanager/blob/master/views/cypress/utils/resource-manager.js
taoQtiItem can be used without taoMediaManager